src/Money.php

```diff
@@ -539,4 +482,28 @@ public function jsonSerialize()
             'currency' => $this->currency,
         );
     }
+
+    /**
+     * @param Calculator[] $calculators
+     */
+    public static function registerCalculators(array $calculators)
```

Comment:
Also, a new calculator should be added to the beggining of the list to make sure custom calculators are checked first.